skyportal/handlers/api/group.py

```diff
@@ -547,6 +547,98 @@ def delete(self, group_id, username):
         return self.success()
 
 
+class GroupUsersFromOtherGroupsHandler(BaseHandler):
+    @auth_or_token
+    def post(self, group_id, *ignored_args):
```

Comment:
Is ignored args needed?

Reply:
Removed

Reply:
This was needed -- errors if removed. Added back